test/test_case_curl_cmd.py

```diff
@@ -0,0 +1,12 @@
+from test.apps._fastapi.app import app
+
+import schemathesis
+from schemathesis import Case
+
+schema = schemathesis.from_dict(app.openapi())
+
+
+@schema.parametrize()
+def test_basic_curl_gen(case: Case):
+    res = case.as_curl_command()
+    assert res == "curl -X GET " "-H 'User-Agent: schemathesis/2.6.0' " "http://localhost/users"
```

Comment:
```suggestion
    command = case.as_curl_command()
    assert command == "curl -X GET -H 'User-Agent: schemathesis/2.6.0' http://localhost/users"
```

Reply:
There is also `schemathesis.constants.USER_AGENT` constant that could be used to format the expected string to avoid this test breaking after a version bump